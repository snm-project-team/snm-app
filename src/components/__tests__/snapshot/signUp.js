import React from 'react';
import renderer from 'react-test-renderer';
import SignUp from '../../signUp';

it('renders without crashing', () => {
  const rendered = renderer.create(
    <SignUp
      signUp={() => {}}
    />,
  ).toJSON();
  expect(rendered).toMatchSnapshot();
});
