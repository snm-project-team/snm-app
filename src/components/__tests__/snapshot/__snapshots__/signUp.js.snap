// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  enabled={true}
  keyboardVerticalOffset={0}
  onLayout={[Function]}
  style={
    Object {
      "marginBottom": 20,
      "marginTop": 100,
      "padding": 30,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="メールアドレス"
    placeholderTextColor="#7c7c7c"
    style={
      Object {
        "borderWidth": 1,
        "marginBottom": 20,
      }
    }
    underlineColorAndroid="rgba(0,0,0,0)"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="パスワード"
    placeholderTextColor="#7c7c7c"
    style={
      Object {
        "borderWidth": 1,
        "marginBottom": 20,
      }
    }
    underlineColorAndroid="rgba(0,0,0,0)"
    value=""
  />
  <View
    accessibilityComponentType="button"
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        登録
      </Text>
    </View>
  </View>
</View>
`;
